src/components/LoginPage/index.js

```diff
@@ -19,17 +19,31 @@ class LoginPage extends React.Component {
       email: '',
       password: '',
       loading: false,
+      error: '',
       feedbackMessage: ''
     };
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.validateEmail = this.validateEmail.bind(this);
   }
 
   handleChange(e) {
     this.setState({
       [e.target.name]: e.target.value
     });
+    this.error = this.state;
+    if (this.error) {
+      this.setState({
+        error: ''
+      });
+    }
+  }
+
+  validateEmail(email) {
+    // eslint-disable-next-line no-useless-escape
+    const emailRegEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
@MutegekiHenry this is super excellent stuff for senior devs, I commend you on this.